StayFit/TabBarController/ProfileView/ProfileViewController.m

```diff
@@ -137,4 +138,44 @@ -(void) didTapPhoto {
     [self presentViewController:imagePickerVC animated:YES completion:nil];
 }
 
+//allows user to send email
+- (IBAction)contact:(id)sender {
+    if ([MFMailComposeViewController canSendMail])
+    {
+        MFMailComposeViewController *mail = [[MFMailComposeViewController alloc] init];
+        mail.mailComposeDelegate = self;
+        [mail setSubject:@"Sample Subject"];
+        [mail setMessageBody:@"Here is some main text in the email!" isHTML:NO];
+        [mail setToRecipients:@[@"nikitaag206@gmail.com"]];
+
+        [self presentViewController:mail animated:YES completion:NULL];
+    }
+    else
+    {
+        NSLog(@"This device cannot send email");
+    }
+}
+
+- (void)mailComposeController:(MFMailComposeViewController *)controller didFinishWithResult:(MFMailComposeResult)result error:(NSError *)error
+{
+    switch (result) {
```

Comment:
nice use of switch cases. Ideally we would want to remove extra spaces

Reply:
> nice use of switch cases. Ideally we would want to remove extra spaces

i'll remove the spaces and merge then